crates/parol-ls/src/parol_ls_grammar_trait.rs

```diff
@@ -11,14 +11,12 @@
 
 use parol_runtime::derive_builder::Builder;
 use parol_runtime::id_tree::Tree;
-#[allow(unused_imports)]
-use parol_runtime::parol_macros::{pop_and_reverse_item, pop_item};
-
 use parol_runtime::log::trace;
 #[allow(unused_imports)]
 use parol_runtime::miette::{bail, miette, IntoDiagnostic, Result};
+#[allow(unused_imports)]
+use parol_runtime::parol_macros::{pop_and_reverse_item, pop_item};
 use parol_runtime::parser::{ParseTreeStackEntry, ParseTreeType, UserActionsTrait};
-
```

Comment:
This change was inevitable. The output is identical for all crates except for `parol-ls`.